src/gui/widgets/EnvelopeAndLfoView.cpp

```diff
@@ -590,8 +518,7 @@ void EnvelopeAndLfoView::lfoUserWaveChanged()
 		if( m_params->m_userWave.frames() <= 1 )
 		{
 			TextFloat::displayMessage( tr( "Hint" ),
-				tr( "Drag a sample from somewhere and drop "
-					"it in this window." ),
+				tr( "Drag and drop a sample in this window." ),
```

Comment:
How about "Drag and drop a sample **into** this window"? Because, correct me if I'm wrong, but I don't think you can drag a sample _out_ of it.

Reply:
You're correct. I'll change it.